### ISSUE TITLE
Add support to @author tag and others;

### ISSUE THREAD

**Issue by GCambraia** [madhavd1/vscode-javadoc-tools#21]
Would be amazing if you could add support to other tags such as @author, @date or @version. Just to get a closer result as to what you would get from eclipse IDE's javadoc, for example.

**Comment by noel-buergler** [comment]
Yeah would be amazing. I also would like to have them. Especially `@author` and `@version`.
